Nosenti/Innocent-Ingabire-portfolio
# Ft blog

#### What does this PR do?
This pull request restructure my folders, add blog, portfolio pages and responsiveness of all the pages. It also adds figma designs.

#### Description of Task to be completed
Completed design and development of all pages. By now, I assume to have completed the entire project for this sprint unless I get feedback and twist a few things.
#### How should this be manually tested?
You can clone/download the project folder and navigate inside it. You will see the folders which have different pages and you can click on them to see HTML, css and figma files. Click on the HTML file and it will open in the browser.
To access other pages, you click on them on the navigation bar.
You can resize the browser window to test the responsiveness of the pages.

#### Any background context you want to provide?
**Note:** 
Sign in page is not linked to the rest and here is my rationale:
Since It is only used by admin/blog owner, there is no reason to put it on other pages, that would prompt the readers to click on it. Second, It does not have the navigation bar since it will not be available through other pages, it will only be available through Url and upon signing, you will land on the landing page with Admin privileges and you will access other pages as the admin not the reader.

#### What are the relevant stories?
<blockquote class="trello-card"><a href="https://trello.com/c/P3YP0CRX/16-blog">Blog</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>

<blockquote class="trello-card"><a href="https://trello.com/c/Y8lANTT5/4-create-article">Create article</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>

#### Screenshots (if appropriate)
![Blog Page-Admin](https://user-images.githubusercontent.com/36986840/88484128-ccf13980-cf6c-11ea-86a9-8b1a13521c3f.jpg)
![Portfolio](https://user-images.githubusercontent.com/36986840/88484132-d8446500-cf6c-11ea-8a00-9958d4375aa8.jpg)
